src/components/widgets/HiddenWidget.js

```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from "react";
+
+
+function HiddenWidget({
+  schema,
+  placeholder,
+  value,
+  defaultValue,
+  required,
+  onChange
+}) {
+  return (
+    <input type="hidden"
+      value={value}
+      defaultValue={defaultValue}
+      placeholder={placeholder}
+      required={required}
+      onChange={(event) => onChange(event.target.value)} />
```

Comment:
good question... I thought about it. Maybe in combination with [_Add computed fields ? #45_](https://github.com/mozilla-services/react-jsonschema-form/issues/45).
But I think it's safe to say that the placeholder won't be needed.